src/main/java/org/kumoricon/registration/model/staff/BadgeResourceService.java

```diff
@@ -78,19 +78,13 @@ public BadgeResource getBadgeResourceForWithBackground(BadgeType badgeType, Bool
         }
     }
 
-    private void setAttendeeBackground(BadgeType badgeType, Boolean withAttendeeBackground) {
+    private BadgeResource getAttendeeBadgeResourcesWithBackgroundOverride(BadgeType badgeType, Boolean withAttendeeBackground) {
         log.info("Overriding current printAttendeeBackground setting of {} to {}", printAttendeeBackgrounds, withAttendeeBackground);
-        Map<Boolean, PDDocument> backgroundFuncMap = Map.of(
-                true, loadBackground("Print - Kumoricon-2021-Badge-" + badgeType + ".pdf"),
-                false, buildBlankAttendeeBadge()
-        );
 
-        switch (badgeType) {
-            case ATTENDEE -> attendeeBadgeBackground = backgroundFuncMap.get(withAttendeeBackground);
-            case VIP -> vipBadgeBackground = backgroundFuncMap.get(withAttendeeBackground);
-            case SPECIALTY -> specialtyBadgeBackground = backgroundFuncMap.get(withAttendeeBackground);
-            default -> log.warn("Tried to set attendee badge background for type {} that is not supported in BadgeResourceService", badgeType);
-        }
+        PDDocument background = withAttendeeBackground ?
+                loadBackground("Print - Kumoricon-2021-Badge-" + badgeType + ".pdf") : buildBlankAttendeeBadge();
```

Comment:
Instead of loading it again, just return attendeeBadgeBackground (since that gets loaded when the service starts up)

Also, that saves changing the filename for the year in two different places, which someone [me] is almost certain to forget to do.(In fact, it should probably just be "badge-attendee.pdf", "badge-vip.pdf", etc.)

You could probably combine all this in the switch statement around line 64, with something like:
```
case ATTENDEE:
    return withBackground ? attendeeBadgeBackground : buildBlankAttendeeBadge();
etc...
```

As I was typing all this out, it occurs to me that loading the backgrounds from a PDF for attendees is actually only used for testing/laying out the badge, and when we actually print them in production it's always a blank background, since the badge shells are pre-printed.

Staff/Guest badges always get the background printed, though - those are just blank pages in the printer.

Reply:
In fact, if there's a checkbox for it there and on the test page, could probably remove the badge.printAttendeeBackgrounds setting entirely

Reply:
added a function to return BadgeResources with blank attendee badge, also removed the printAttendeeBackgrounds setting in my last commit, but if we don't want to do that just yet i can revert it

Reply:
Nope, looks great to me. The printAttendeeBackgrounds setting was really just a hack for when I was generating badges, and since that happens a couple times a year making it more configurable was never high on the priority list.